model_templates/training/python3_keras_vizai_joblib/README.md

```diff
@@ -0,0 +1,11 @@
+## Python Keras Visual AI Training Model Template
```

Comment:
please elaborate on how the CSV file works here.

Reply:
Actually with DR app, both `.csv` and `.zip` should work well (as DSS converts zip to csv internally). iow for img classification problem, when images are grouped under directories based on the labels and zipped, DSS will convert them to csv for the model to process. Here in this example, I've just used readily available s3://datarobot-private-image-datasets/cats_dogs_small_train.csv (zip version is here s3://datarobot-private-image-datasets/cats_dogs_small.zip)